src/ui/developerMenu/ViewDeveloperMenu.qml

```diff
@@ -72,9 +71,27 @@ VPNFlickable {
         imageRightSrc: "../resources/chevron.svg"
         onClicked: stackview.push("../developerMenu/ViewFeatureList.qml")
     }
+    VPNExternalLinkListItem {
+        id:inspectorLink
+        visible: stagingServer.isChecked && !restartRequired.visible
```

Comment:
I think the intention here is that we only want the link to be visible when the inspector is actually running, which we can get directly with `VPN.stagingMode`. Making use of `restartRequired.visible` leads to a bit of a convoluted dependency graph.